### PR TITLE
UI/profile

### PR DESCRIPTION
**Pull request by Akshay-Jak**
### Functionality:
Migrated the UI 

### Solution:
Changed the UI with given design

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

File: src/app/app.component.css [hunk 1]
```diff
@@ -35,6 +35,8 @@ body {
     --button-text2: #4285F4;
     --table-header: #545454;
     --scrollbar-color: rgba(84, 84, 84, 0.17);
+    --togglebutton-color: #f3d4d6;
+    --text-color: #ccc4c4;
```
Comment (viv9k) [inline src/app/app.component.css:39]: Use Global variables for test color, and button toggle color

File: src/app/app.component.css [hunk 2]
```diff
@@ -64,4 +66,6 @@ body {
     --button-text2: #4285F4;
     --table-header: #d4d4d6;
     --scrollbar-color: rgba(255, 255, 255, 0.51);
+    --togglebutton-color: #29382c;
+    --text-color: #ccc4c4;
```
Comment (viv9k) [inline src/app/app.component.css:70]: Use Global variables for test color, and button toggle color

**Review by viv9k — APPROVED** [review]
Approved

**Review by Pendyala-thanvi — COMMENTED** [review]
Resolve merge conflicts.

File: src/app/app.component.css (hunk 1 above)
Comment (Akshay-Jak) [reply src/app/app.component.css:39]: Resolved changes

File: src/app/app.component.css (hunk 2 above)
Comment (Akshay-Jak) [reply src/app/app.component.css:70]: Resolved changes

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**